public/css/nhsuk-ie6.css

```diff
@@ -1,28 +1,28 @@
 @charset "UTF-8";
 @font-face {
   font-family: "FS Me";
-  src: url("/* @echo FONT_CDN_PATH */assets/fonts/fsme/FSMeWeb-Light.eot?") format("eot"), url("/* @echo FONT_CDN_PATH */assets/fonts/fsme/FSMeWeb-Light.woff") format("woff");
+  src: url("https://dhrlmnmyf2njb.cloudfront.net/assets/fonts/fsme/FSMeWeb-Light.eot?") format("eot"), url("https://dhrlmnmyf2njb.cloudfront.net/assets/fonts/fsme/FSMeWeb-Light.woff") format("woff");
```

Comment:
This seems at odds with the outcome of the slack conversation but I'm sure I've misinterpreted it:

> Yep. We only have a license for certain domains
> 
> [5:56]  
> That domain is just our cloudfront cdn which sits in front of an s3 bucket
> 
> [5:58]  
> Also, it's best not to reference assets on beta as they're all revved and sitting on a cdn
> 
> [5:58]  
> If you need compiled assets take a copy and serve locally